vizro-core/docs/pages/user-guides/customizing-css.md

```diff
@@ -0,0 +1,351 @@
+                vm.Card(text="""Lorem ipsum dolor sit amet consectetur adipisicing elit.""")
+                     ],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+
+        Vizro().build(dashboard).run()
+        ```
+
+        <img src=https://py.cafe/logo.png alt="py.cafe logo" width="30"><b><a target="_blank" href="https://py.cafe/vizro-official/vizro-custom-card-styling">Run and edit this code in Py.Cafe</a></b>
+
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to add data to the data manager and parse YAML configuration
+        # See yaml_version example
+        pages:
+        - components:
+            - text: |
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              type: card
+              id: custom-card
+            - text: |
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              type: card
+          title: Changing the card color
+        ```
+    === "Result"
+         [![CardCSS]][CardCSS]
+
+    [CardCSS]: ../../assets/user_guides/assets/css_change_card.png
+
+
+Some Vizro components produce a single HTML element with an ID that matches the model ID, allowing you to target it
+directly using the #id selector in CSS. Other components generate multiple HTML elements. Within these, the "core"
+element will have an ID matching the model ID, while non-core elements may have IDs that are variations of it,
+such as `{model-id}-title`.
+
+In all instances, you can determine the correct selector by using Chrome DevTools or a similar tool after setting the
+appropriate model ID.
+
+
+## Common examples
+
+### Turn off page title
+See the example above on [hiding the page title on selected pages](#overwrite-css-for-selected-pages).
+
+### Change the background color of a card
+See the example above on [customizing CSS properties in selective components](#overwrite-css-for-selected-components).
+
+### Change the font
+The default fonts for a Vizro app are `Inter, sans-serif, Arial, serif`.
+
+If you need to change the global font, perhaps to adhere to branding guidelines, follow these steps:
+
+1. Download the desired font from a font provider such as [Google Fonts](https://fonts.google.com/).
+2. Place the font file (`.ttf`, `woff2`, etc.) into your `assets` folder. Here’s an example of what the assets folder might look like:
+
+    ![Font Change](../../assets/user_guides/customizing_css/font-change.png)
+3. Add the font to your CSS file using the `@font-face` rule and apply the font globally to your Vizro app, making sure
+to specify fallback fonts. Add the following to your `custom.css` file:
+
+
+    ```css
+    @font-face {
+      font-family: PlayfairDisplay;
+      src: url("PlayfairDisplay-VariableFont_wght.ttf") format("truetype");
+    }
+
+    * {
+      font-family: PlayfairDisplay, Inter, sans-serif, Arial, serif;
+    }
+    ```
+
```

Comment:
I wanted to add a section here that one has to separately replace the font inside the charts as well, but it didn't seem to work as I expected it (see scratch dev)